scipy/stats/_mstats_basic.py

```diff
@@ -2551,7 +2551,7 @@ def _winsorize1D(a, low_limit, up_limit, low_include, up_include,
                                    upinc, contains_nan, nan_policy)
 
 
-def moment(a, moment=1, axis=0):
+def moment(a, moment=1, central=None, axis=0):
```

Comment:
We must respect the current order to not break backward compatibility. Also we prefer to add new parameters as keywords only.
```suggestion
def moment(a, moment=1, axis=0, *, central=None):
```

Reply:
Thanks for the suggestion, committed!

Reply:
Thanks! For later, please don't resolve conversations yourself and don't force push. This helps reviewers to keep track of the status.